lib/schema.js

```diff
@@ -38,7 +38,8 @@ var express = require("express"),
     RemoteAccessToken = require("./model/remoteaccesstoken").RemoteAccessToken,
     RemoteRequestToken = require("./model/remoterequesttoken").RemoteRequestToken,
     ActivityObject = require("./model/activityobject").ActivityObject,
-    DatabankStore = require('connect-databank')(express),
+    session = require("express-session"),
+    DatabankStore = require('connect-databank')(session),
```

Comment:
@profOnno happy to do the review.

I pushed a `.jscsrc` to the repo; if you merge `master` into your branch you should get it.